docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -542,6 +542,16 @@ The name of the test suite, which will be reflected in the results and related a
 ```
 ---
 
+### `testApp`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Set test application on suite level. See the full [usage](#testapp). If this field is not set, saucectl will pick the default `testApp` from [espresso](#espresso) level.
+
+```yaml
+  testApp: ./apps/calc-success.apk
```

Comment:
Would be good to put the parent context (suite) in here as well. Makes it easier to see where this attribute belongs. If you recall, the idea was to make those changes incrementally, which should make the docs easier to use as reference material :)

Reply:
make sense.